components/Image.html

```diff
@@ -36,7 +36,49 @@
                 </ul>
             </aside>
             <div class="main__grid-item">
-               
+                <div class="component">
+                    <p class="component__heading">Responsive Images</p>
+                    <p class="component__text">Responsive Image Components are used to display Images 
+                        on websites, which are screen responsive. They expand upto the width of the 
+                        container they are in.</p>
+                    <p class="component__heading-primary">Round Images</p>
+                    <p class="component__sub-text">
+                         
+                    </p> 
+                    <div class="component__view avatar__comp">
+                        <div class="round__image-div">
+                            <img src="/assets/images/Pika.jpeg" alt="" class="round__img"/>
+                        </div> 
+                        <div class="round__image-div">
+                            <img src="/assets/images/spiderman.jpg" alt="" class="round__img"/>
+                        </div>  
+                    </div> 
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/3cvvSiCBwO4m9mcG082p"
+                        style="width: 740px; height: 294px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                        </iframe>
```

Comment:
instead of giving inline styles here, you can make one class for code snippets and you can use it for every code snippet